js/task2.js

```diff
@@ -0,0 +1,24 @@
+function greatestCommonFactor() {
+    let NOD;
+    const a = prompt('Введите первое число: ');
+    const b = prompt('Введите второе число: ');
+
+    let res = (a.replace(/\d+/, '') || b.replace(/\d+/, ''));
```

Comment:
You can also simplify with `+prompt`. In that case you will avoid extra line of code for RegExpp